### PR TITLE
[[ Gradient Ramp Editor ]] Add selected stop API

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: extensions/widgets/gradientrampeditor/notes/feature-selected_stop.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+## selectedStop property
+The gradient ramp widget now has a selectedStop property whose value is
+the index of the widget's selected gradient stop, or 0 if there is no
+selected stop.
+
+## selectedStopChanged message
+The gradient ramp widget posts a selectedStopChanged message when the 
+selected stop of the widget is changed either by script (via the 
+selectedStop property) or by user interaction,, i.e. clicking on one 
+of the widget's gradient stops.
```
Comment (peter-b) [inline extensions/widgets/gradientrampeditor/notes/feature-selected_stop.md:12]: Two commas ",,"!

Do you think it's worth making the newly added identifiers **bold** to distinguish them?

Comment (livecodeali) [inline extensions/widgets/gradientrampeditor/notes/feature-selected_stop.md:12]: What can I say,, I just love those commas.
Yes, bold identifiers is a good call.

**Comment by livecodeali** [comment]
@peter-b updated!

**Comment by peter-b** [comment]
@livecode-vulcan review ok a1e9687

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok a1e96878f9507b53aa4b203c3e87fb0e24da5242

**Comment by livecode-vulcan** [comment]
😎 test success a1e96878f9507b53aa4b203c3e87fb0e24da5242
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1644)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1660)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1639)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1650)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1661)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1663)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1668)

**Pull request merged**